SecretManagement.KeePass/SecretManagement.KeePass.Extension/Public/Get-Secret.ps1

```diff
@@ -8,19 +8,19 @@ function Get-Secret {
     if ($AdditionalParameters.Verbose) {$VerbosePreference = 'continue'}
 
     if (-not (Test-SecretVault -VaultName $vaultName -AdditionalParameters $AdditionalParameters)) {
-        Write-Error 'There appears to be an issue with the vault (Test-SecretVault returned false)'
+        Write-PSFMessage -Level Error 'There appears to be an issue with the vault (Test-SecretVault returned false)'
```

Comment:
This now fails the throw test. I think it is because these errors are trapped in a specific way in order to be emitted correctly